contracts/modules/Payments.sol

```diff
@@ -45,12 +45,16 @@ abstract contract Payments is RouterImmutables {
         if (token == Constants.ETH) {
             uint256 balance = address(this).balance;
             uint256 amount = (balance * bips) / FEE_BIPS_BASE;
-            recipient.safeTransferETH(amount);
+            uint256 fixedAmount = (amount * PAYMENT_AMOUNT_BIPS) / FEE_BIPS_BASE;
```

Comment:
do we want to handle case when this these are unset and `fixedAmount` is 0? I.e if contract is initialized with `PAYMENT_AMOUNT_BIPS` is 0 and/or `PAYMENT_RECIPIENT` is addr 0? Would save a call

Reply:
I think in that case we just wouldnt deploy with this feature. it wont be modifiable so if we want `PAYMENT_AMOUNT_BIPS==0` we would remove this code entirely